nomic/tests/test_atlas_client.py

```diff
@@ -468,3 +468,10 @@ def test_map_text_iterator():
     map = project.get_map(name='UNITTEST_pandas_text')
     assert project.total_datums == 50
     project.delete()
+
+
+def test_list_projects():
+    projects = atlas.list_projects()
```

Comment:
several of the other tests create projects, so I figure this one should also temporarily create a project if its testing something that will only work if at least one exists - it doesn't need to bother with creating an _index_

Reply:
Addressed